src/Matrix.cc

```diff
@@ -2492,6 +2493,18 @@ NAN_METHOD(Matrix::MeanWithMask) {
   info.GetReturnValue().Set(arr);
 }
 
+NAN_METHOD(Matrix::Mean) {
+  SETUP_FUNCTION(Matrix)
+
+  cv::Scalar means = cv::mean(self->mat);
+  v8::Local<v8::Array> arr = Nan::New<Array>(3);
```

Comment:
Means always return a 3 element scalar irrespective of 1-3 channels in an image. If there is only 1 channel for example then the first element in the only valid value.